lib/models/categories/categories-collection.js

```diff
@@ -0,0 +1,32 @@
+const categorySchema = require('./categories-schema.js'); 
+
+const Categories = require('../mongo.js');
+
+class Categories extends Model {
+    constructor() {
+        super(schema);
```

Comment:
the schema should be call categorySchema because that what your referencing the categories schema